.gitignore

```diff
@@ -60,3 +60,6 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+work/
```

Comment:
Let me know if you are ok with this, or if we should discuss and plan out the directory structure more.